gcovr/tests/html-source-encoding-cp1252/Makefile

```diff
@@ -1,5 +1,5 @@
 all:
-	$(CXX) -fprofile-arcs -ftest-coverage -fPIC -finput-charset=cp1252 main.cpp -o testcase
+	$(CXX) -fprofile-arcs -ftest-coverage -fPIC $(if $(patsubst clang++%,,$(CXX)),-finput-charset=cp1252,) main.cpp -o testcase
```

Comment:
If Clang just ignores alternative source encodings, it might be better to skip this test. After all, this test is more about our HTML rendering than the compiler or the gcov tool. Currently, test skipping has to be configured in the `test_gcovr` file.

Reply:
Ok, I'll skip the test.